### ISSUE TITLE
Bind unbound values at the toplevel

### ISSUE THREAD

**Issue by ihodes** [diml/utop#161]
E.g. 

``` ocaml
> Uri.parse "some uri";;
> Uri.port _1;;
```

**Comment by diml** [comment]
Is that what you are looking for?

```ocaml
let x = Uri.parse "some uri";;
```

**Comment by ihodes** [comment]
No, I'm aware of let-bindings :) 

Other REPLs have this feature (e.g. iPython (`_1`, `_2`, etc.) and Clojure (`*1`, `*2`, etc). It comes in handy a lot in my experience, and saves a good amount of typing in general.

**Comment by diml** [comment]
I see. That's seems like a nice feature indeed

**Comment by diml** [comment]
Implemented by #200
